### DISCUSSION TITLE
Where exactly are the recipes stored?

### DISCUSSION THREAD

**Discussion by SoleInvictus** [TandoorRecipes/recipes#2739] [Q&A]
I'm running Tandoor in docker via community apps in Unraid. My original install got borked for reasons I couldn't figure out, so I copied the recipes.db file, deleted the docker, and reinstalled. I then noticed the new 'blank' recipes.db file was of identical size - 1.64MB. Opening it up, the new and old .db files are identical.

Where exactly are the recipes being stored? All I have for the docker in appdata is the recipes/data and recipes/media/recipes folders. Pictures clearly get added in the latter but I'm stumped otherwise.

**Comment by smilerz** [comment]
It would either be postgres or db.sqlite
